code/__HELPERS/bygex/bygex.dm

```diff
@@ -48,11 +48,13 @@
 	regex_replaceall(str, exp, fmt)
 		return call(LIBREGEX_LIBRARY, "regex_replaceall")(str, exp, fmt)
 
+#if DM_VERSION < 510
 	replacetextEx(str, exp, fmt)
```

Comment:
Not sure if the game works if you turn off BYGEX and use the old procs, but done nevertheless.